README.md

```diff
@@ -269,6 +287,26 @@ var newArray = arrayElement.filter(function(item) {
 }); // newArray.toValue() is now ['a']
 ```
 
+##### reduce
+
+The `reduce` method may be used to reduce over a Minim array or object. The method takes a function and an optional beginning value.
+
+```javascript
+var numbers = new minim.ArrayElement([1, 2, 3, 4]);
+var total = numbers.reduce(function(result, item) {
+  return result.toValue() + item.toValue();
+}); // total.toValue() === 10
+```
+
+The `reduce` method also takes an initial value, which can either be a value or Minim element.
+
+```javascript
+var numbers = new minim.ArrayElement([1, 2, 3, 4]);
+var total = numbers.reduce(function(result, item) {
+  return result.toValue() + item.toValue();
+}, 10); // total.toValue() === 20
+```
```

Comment:
I am asking this because not many know the standard for the parameters sent to the callback function for an `object.reduce`